XDescription: <short summary of the patch>
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 arxivupdates (1.0-1) unstable; urgency=medium
 .
   * Initial release (Closes: )
Author: Gorla Praveen <gorlapraveen@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2018-10-08

--- arxivupdates-1.0.orig/arxiv_required
+++ arxivupdates-1.0/arxiv_required
@@ -4,5 +4,4 @@ echo 'Installing required packages for A
 echo '-------------------------------------------------------'
 bash -c "sudo pip3 install -r requirements.txt"
 python3 /usr/bin/requirements.py
-python3 /usr/local/bin/requirements.py
 
